spec/models/education_benefits_claim_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "rails_helper"
+
+RSpec.describe EducationBenefitsClaim, type: :model do
+  let(:attributes) do
+    {
+      form: { foo: true }
+    }
+  end
+  subject { described_class.new(attributes) }
+
+  describe "validations" do
+    it { should validate_presence_of(:form) }
```

Comment:
hmmm, i think they used to just verify the macro. I still don't know if it's necessary to introduce a whole other testing framework on top of rspec but, if it makes testing easier / faster, I'm all for that. 

Why wouldn't you also want to test validations on the other attributes that have validations? Or if form is an association, it could be tested independently and then you just stub the test for form to always return true or false.

Shoulda matchers just add a lot of extra cruft, introduce its own sort of mechanism for testing and once it's introduced to a project, people start using its "convenience methods" without considering the rspec way of doing things using stubs and doubles instead (faster and better practice).